webapps-backup/src/main/java/io/camunda/webapps/backup/repository/BackupRepositoryProps.java

```diff
@@ -21,7 +21,9 @@ public interface BackupRepositoryProps {
 
   Long incompleteCheckTimeoutInSeconds();
 
-  boolean includeGlobalState();
+  default boolean includeGlobalState() {
```

Comment:
This setting was true for tasklist & false for operate, I guess this should be true by default